fr.centralesupelec.edf.riseclipse.developer.p2_to_m2/src/main/resources/README.md

```diff
@@ -0,0 +1,22 @@
+Copyright (c) 2021 CentraleSupélec & EDF.
+All rights reserved.\
+This program and the accompanying materials
+are made available under the terms of the Eclipse Public License v2.0
+which accompanies this distribution, and is available at
+https://www.eclipse.org/legal/epl-v20.html
+
+This file is part of the RiseClipse tool.
+
+### Contributors
+
+Computer Science Department, CentraleSupélec\
+EDF R&D
+
+### Contacts
+
+dominique.marcadet@centralesupelec.fr\
+aurelie.dehouck-neveu@edf.fr
+
+### Web site
+
+https://riseclipse.github.io
```

Comment:
@marcadetd Here is the readme that you requested to replace the copyrights comment in the aggr file

Reply:
Put it in the p2_to_m2 folder and add a sentence about the use of CBI/aggregator tool to edit the file (with the link I have given)